iterfzf/__init__.py

```diff
@@ -59,6 +60,11 @@ def iterfzf(
         cmd.append('--multi')
     if not mouse:
         cmd.append('--no-mouse')
+    if bind:
+        bind_options = ','.join(
+            [r"{}:{}".format(key, action) for key, action in bind.items()]
```

Comment:
I believe we can use just a generator expression instead of a list comprehension here.

```suggestion
            r"{}:{}".format(key, action) for key, action in bind.items()
```

Reply:
Sure, I missed that. We only iterate once just to create the --bind format. It's more efficient to use a generator expression.